Library/Homebrew/formula.rb

```diff
@@ -358,7 +358,14 @@ def update_head_version
     return unless head.downloader.is_a?(VCSDownloadStrategy)
     return unless head.downloader.cached_location.exist?
 
-    head.version.update_commit(head.downloader.last_commit)
+    path = ENV["PATH"]
+    unless path.include?("/opt/")
```

Comment:
That will inadvertently match MacPorts default location, too, no?

Reply:
@ilovezfs Changed the approach.